vmdk_plugin/drivers/vmdk/vmdk_driver.go

```diff
@@ -251,6 +252,12 @@ func (d *VolumeDriver) Create(r volume.Request) volume.Response {
 	if errAttach != nil {
 		log.WithFields(log.Fields{"name": r.Name,
 			"error": errAttach}).Error("Attach volume failed, removing the volume ")
+		if strings.Contains(errAttach.Error(), recvError) {
```

Comment:
This is actually an unspecified error from the VMCI client itself. The VMCI client is returning from a recv() call without actually waiting for  data from the vSphere service. Errno isn't set and hence current error handling code  doesn't catch this condition.

And without error codes in use only error messages are passed to the volume driver and need to check for strings vs. an error code.

Unless we change the vmdkops interface to return error code in addition to error message?